sdk/core/Azure.Core.Experimental/src/DynamicData/DynamicCaseMapping.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.Dynamic
+{
+    /// <summary>
+    /// Options for getting and setting DynamicData properties.
+    /// </summary>
+    public enum DynamicCaseMapping
+    {
+        /// <summary>
+        /// Properties are read from and written to the data buffer with the same casing as the DynamicData property.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// A "PascalCase" DynamicData property can be used to read and set "camelCase" properties that exist in the data buffer.
```

Comment:
Thank you for casing the names as such for those less familiar with the casing conventions! 🙏

Reply:
👍